packages/meditrak-server/src/routes/createUser.js

```diff
@@ -24,8 +24,9 @@ const PERMISSION_GROUPS = {
 
 const DEMO_LAND_NAME = 'Demo Land';
 
+//Used for registering user account
 export const createUser = async (req, res) => {
-  await req.assertPermissions(assertBESAdminAccess);
+  await req.assertPermissions(allowNoPermissions);
```

Comment:
This endpoint is actually for registering user account from Tupaia webpage so I think this should actually be for "anyone"